helm/dagger/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: dagger-helm
+description: Dagger Helm chart
+
+type: application
+version: 0.0.2
+appVersion: v0.8.4
```

Comment:
Are we thinking of introducing `./hack/make helm:bump` which will bump `appVersion` same as `sdk:go:bump` bumps the Engine version in the Go SDK?

Reply:
I hadn't considered that, but I like the idea. 😄 

I'll take a look at `sdk:go:bump` and pull some code from it.